README.md

```diff
@@ -29,7 +29,6 @@ This is because VS Code has not implemented those features. Head on over to this
 Additionally, you can install an extension for many of these features:
 
 * [FontSize Shortcuts](https://marketplace.visualstudio.com/items?itemName=peterjuras.fontsize-shortcuts)
-* [change case](https://marketplace.visualstudio.com/items?itemName=wmaurer.change-case)
 * [transpose](https://marketplace.visualstudio.com/items?itemName=v4run.transpose)
```

Comment:
is there a reason to remove the change case extension?

Reply:
Actually I just removed because this PR adds the feature we had on atom to convert case. If you prefer, I can just keep this extension on list.

Reply:
the change case extension adds a bunch of different transformations, so there is no reason to remove it from here.

Reply:
i can just deal with it after merging the pr, so no worries.